ca_gazebo/include/ca_gazebo/create_cliff_plugin.h

```diff
@@ -0,0 +1,78 @@
+#ifndef GAZEBO_ROS_CLIFF_SENSOR_HH
+#define GAZEBO_ROS_CLIFF_SENSOR_HH
+
+#include <ros/ros.h>
+#include <ros/advertise_options.h>
+#include <tf/tf.h>
+#include <tf/transform_listener.h>
+
+#include <vector> 
+
+#include <gazebo/transport/transport.hh>
+#include <gazebo/plugins/RayPlugin.hh>
+
+#include <gazebo_plugins/gazebo_ros_utils.h>
+#include <gazebo_plugins/PubQueue.h>
+#include <std_msgs/Bool.h>
+
+namespace gazebo
+{
+  class GazeboRosCliff : public RayPlugin
+  {
+    /// \brief Constructor
+    public: GazeboRosCliff();
```

Comment:
In this class, you're using a different variable format. Maintain the format across the classes.

Reply:
@eborghi10 I'm not understanding well this comment

Reply:
Don't use this format:

```C++
public: ...
public: ...
public: ...
```